_shared/project/conf/supervisord-dev.conf

```diff
@@ -19,6 +19,9 @@ silent=true
 {{ program(program_name, program_settings) }}
 {% endfor %}
 {% else %}
+{% if cookiecutter._directory == "pyramid-app" and cookiecutter.get("postgres") == "yes" %}
+{{ program("make_db", {"command": "bin/make_db", "startsecs": "0"}) }}
+{% endif %}
```

Comment:
Add the `make_db` process to `make dev`'s Supervisor config (only for Pyramid apps with `postgres=yes`).